### PR TITLE
Allow setting an alternate manifest source in your user-data

### PR DESCRIPTION
**Pull request by DeviaVir**
This allows you to set `#manifest: /path/to/manifest` anywhere in your user-data (no need to use JSON) after which we'll attempt to load that file and parse it as is.
Especially useful for "cloud-config" or simple bash user-data scripts.

Fixes https://github.com/mycloudandme/spacel-agent/issues/2

@pebble/webops

### CODE REVIEW

File: src/spacel/aws/meta.py [hunk 1]
```diff
@@ -17,11 +17,20 @@ def __init__(self):
...
         self.region = self.az[:-1]
-        self.user_data = json.loads(self._get(USER_DATA_URL, '{}'))
+        self.user_data = self._parse_user_data(self._get(USER_DATA_URL, '{}'))
         self.name = self.user_data.get('name', 'test')
         self.orbit = self.user_data.get('orbit', 'test')
         self.bastion = self.user_data.get('bastion', False)
 
+    def _parse_user_data(self, data):
+        split = data.splitlines()
+        lines = [line for line in split if line.startswith('#manifest:')]
```
Comment (thepwagner) [inline src/spacel/aws/meta.py]: Nit: you're collecting all lines that startwith `#manifest:`; but only using the first.

Maybe simple for loop for dat early exit?

**Comment by thepwagner** [comment]
+1

Alternate ideas: read from stdin or as an argument?

File: src/spacel/aws/meta.py (hunk 1 above)
Comment (DeviaVir) [inline src/spacel/aws/meta.py]: This should solve that: https://github.com/pebble/spacel-agent/pull/1/commits/91e400eeafcbb3b08c2d5a3d8f45135661bcb30b#diff-ba367a9b62a85b200005b1274a003202R27

**Comment by thepwagner** [comment]
+1

**Pull request merged**